crates/nu-data/src/value.rs

```diff
@@ -119,6 +119,13 @@ pub fn compute_values(
                 }?;
                 Ok(UntaggedValue::Primitive(Primitive::Filesize(result)))
             }
+            (Primitive::Filesize(x), Primitive::Int(y)) => match operator {
+                Operator::Plus => Ok(UntaggedValue::Primitive(Primitive::Int(x + y))),
+                Operator::Minus => Ok(UntaggedValue::Primitive(Primitive::Int(x - y))),
+                Operator::Multiply => Ok(UntaggedValue::Primitive(Primitive::Int(x * y))),
+                Operator::Divide => Ok(UntaggedValue::Primitive(Primitive::Int(x / y))),
```

Comment:
We'll need to respect the units here since you pattern match and get into the contents inside.

In math, `1kb + 1` doesn't have a meaning, so we should error if they try to do Operator::Plus over them.

The only two of the list that make sense between filesize and int are Operator::Multiply and Operator::Divide

Reply:
I removed the + and - arms